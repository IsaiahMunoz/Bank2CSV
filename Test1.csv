Gas,Murphy USA,Gas for Car,10,11-Feb-25
Hair cut,Ziba Salon,Hair,35,12-Feb-25
Gas,Shell,Gas for Car,14.05,12-Feb-25
Audra Present,Office De,Miscellaneous,18.9,13-Feb-25
Out with Coworkers,Myunga Inc. Austin Tx,Dining,18.39,13-Feb-25
Audra Gift,H-E-B,Groceries,31.53,13-Feb-25
Beer ,The Jackalope South,Alcohol and Bar,5.5,14-Feb-25
Food,P Terrys Stand,Dining,6.17,14-Feb-25
Ciders,Twin Liquors,Alcohol and Bar,11.36,15-Feb-25
Food w ADS,Waterloo Ice House,Dining,20.18,16-Feb-25
Car Issurance ,Faster Payments Zel Renee Munoz ,Car Issurance,190,16-Feb-25
Phone Bill,Faster Payments Zel Renee Munoz ,Cell Phone,51,17-Feb-25
Rent ,Faster Payments Zel Renee Munoz ,Rent,100,18-Feb-25
Valentine Date,Hecho en Mexico,Dining,88.88,16-Feb-25
Energy Drink,Walgreens,Miscellaneous,3.24,16-Feb-25
Food while cleaning,Whataburger,Dining,7.01,16-Feb-25
gas for car,Murphy USA,Gas for Car,10.02,16-Feb-25
Food with ADS,Asia Cafe,Dining,38.66,16-Feb-25
