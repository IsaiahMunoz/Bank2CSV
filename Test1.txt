FEB 16
2025
Asia Cafe
Food & Dining
– $38.66
$2,095.12
FEB 16
2025
Murphy USA
Gas
– $10.02
$2,133.78
FEB 16
2025
Whataburger
Fast Food
– $7.01
$2,143.80
FEB 16
2025
Walgreens
Pharmacy
– $3.24
$2,150.81
FEB 16
2025
Hecho en Mexico
Restaurants
– $88.88
$2,154.05
FEB 16
2025
Faster Payments Zel Renee Munoz Ztcjaxd
Uncategorized
– $341.00
$2,242.93
FEB 16
2025
Waterloo Ice House
Restaurants
– $20.18
$2,583.93
FEB 15
2025
Twin Liquors
Alcohol & Bars
– $11.36
$2,604.11
FEB 14
2025
P Terrys Stand
Restaurants
– $6.17
$2,615.47
FEB 14
2025
The Jackalope South
Restaurants
– $5.50
$2,621.64
FEB 13
2025
H-E-B
Groceries
– $31.53
$2,627.14
FEB 13
2025
Myunga Inc. Austin Tx
Restaurants
– $18.39
$2,658.67
FEB 13
2025
Office De
Office Suppl
– $18.90
$2,677.06
FEB 12
2025
Shell
Fast Food
– $14.05
$2,695.96
FEB 12
2025
Ziba Salon
Hair
– $35.00
$2,710.01
FEB 11
2025
Murphy USA
Gas
– $10.00
$2,745.01