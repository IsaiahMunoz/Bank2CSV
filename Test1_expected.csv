Descr,Merchant,Category,Price,Date
,Murphy USA,Gas for Car,10.00,"FEB 11, 2025"
,Ziba Salon,Miscellaneous,35.00,"FEB 12, 2025"
,Shell,Dining,14.05,"FEB 12, 2025"
,Office De,Shopping,18.90,"FEB 13, 2025"
,Myunga Inc. Austin Tx,Dining,18.39,"FEB 13, 2025"
,H-E-B,Groceries,31.53,"FEB 13, 2025"
,The Jackalope South,Alcohol and Bar,5.50,"FEB 14, 2025"
,P Terrys Stand,Dining,6.17,"FEB 14, 2025"
,Twin Liquors,Alcohol and Bar,11.36,"FEB 15, 2025"
,Waterloo Ice House,Dining,20.18,"FEB 16, 2025"
,Faster Payments Zel Renee Munoz Ztcjaxd,Miscellaneous,341.00,"FEB 16, 2025"
,Hecho en Mexico,Dining,88.88,"FEB 16, 2025"
,Walgreens,Shopping,3.24,"FEB 16, 2025"
,Whataburger,Dining,7.01,"FEB 16, 2025"
,Murphy USA,Gas for Car,10.02,"FEB 16, 2025"
,Asia Cafe,Dining,38.66,"FEB 16, 2025"
